conans/client/packager.py

```diff
@@ -30,8 +30,8 @@ def new_method(pattern, src=""):
     conanfile.copy_bins = wrap(DEFAULT_BIN)
     conanfile.copy_res = wrap(DEFAULT_RES)
     try:
+        conanfile.package_folder = package_folder
```

Comment:
I would like to have a conanfile.build_folder available too. This way I'll be able to read a spec file from build folder before copy the resources. Yes, I can copy the spec to package and then read it from self.package_folder, but  given that now its not lazy, maybe is more natural to have a source directory and a destination folder and be able to copy things by hand if I want.